test/test_docling_doc.py

```diff
@@ -0,0 +1,286 @@
+import yaml
+
+from collections import deque
+
+from docling_core.types.experimental.document import (
+    BasePictureData,
+    BaseTableData,
+    DescriptionItem,
+    DoclingDocument,
+    TableCell,
+    NodeItem,
+    DocItem,
+    TextItem,
+    FloatingItem,
+    KeyValueItem,
+    SectionItem,
+    PictureItem,
+    TableItem,
+    BasePictureData,
+    BaseTableData
+)
+from docling_core.types.experimental.labels import DocItemLabel, GroupLabel
+
+def test_docitems():
```

Comment:
@cau-git added a unit-test, so we can detect if new classes appear or we have added the serialisation of a DocItem subclass